src/pages/Settings/Settings.tsx

```diff
@@ -53,6 +67,13 @@ export const Settings = ({ userId }: SettingsProps) => {
       >
         Data
       </Typography>
+      <Divider sx={{ my: 2 }} />
+      <Switch
+        checked={playSoundOnNewMessage}
+        onChange={handlePlaySoundOnNewMessageChange}
+      />{' '}
+      Play a sound on new message
```

Comment:
I'd like to tweak this wording to be slightly more straightforward:

```suggestion
      Play a sound when a new message is received
```